ocaml/src/discovery.ml

```diff
@@ -75,6 +75,7 @@ type record = {
     ips: string list;
     port :int option;
     tlsPort: int option;
+    useRdma: bool option;
```

Comment:
ok, but it makes our expression of what we see less accurate: we then have no way to express we did not see any `useRdma` field in the json message.